modules/masterbar/tracks-events.js

```diff
@@ -0,0 +1,109 @@
+/*globals JSON, jetpackTracks */
+(function( $, jetpackTracks ) {
+	window._tkq = window._tkq || [];
+
+	var linksTracksEvents = {
+		//top level items
+		'wp-admin-bar-blog'        : 'jetpack_masterbar_my_sites_link_click',
+		'wp-admin-bar-newdash'     : 'jetpack_masterbar_reader_link_click',
+		'wp-admin-bar-ab-new-post' : 'jetpack_masterbar_write_button_click',
+		'wp-admin-bar-my-account'  : 'jetpack_masterbar_my_account_link_click',
+		'wp-admin-bar-notes'       : 'jetpack_masterbar_notifications_link_click'
+	};
+
+	var linksWhitelist = [
+		//my sites - top items
+		'wp-admin-bar-switch-site', 'wp-admin-bar-blog-info', 'wp-admin-bar-site-view', 'wp-admin-bar-blog-stats', 'wp-admin-bar-plan', 'wp-admin-bar-plan-secondary',
+		//my sites - manage
+		'wp-admin-bar-new-page', 'wp-admin-bar-new-page-secondary', 'wp-admin-bar-new-post', 'wp-admin-bar-new-post-secondary', 'wp-admin-bar-comments',
+		//my sites - personalize
+		'wp-admin-bar-themes', 'wp-admin-bar-themes-secondary',
+		//my sites - configure
+		'wp-admin-bar-sharing', 'wp-admin-bar-users-toolbar', 'wp-admin-bar-users-toolbar-secondary', 'wp-admin-bar-plugins', 'wp-admin-bar-plugins-secondary', 'wp-admin-bar-blog-settings',
+		//reader
+		'wp-admin-bar-following', 'wp-admin-bar-following-secondary', 'wp-admin-bar-discover-discover', 'wp-admin-bar-discover-search', 'wp-admin-bar-discover-recommended-blogs', 'wp-admin-bar-my-activity-my-likes',
+		//account
+		'wp-admin-bar-user-info',
+		// account - profile
+		'wp-admin-bar-my-profile', 'wp-admin-bar-account-settings', 'wp-admin-bar-billing', 'wp-admin-bar-security', 'wp-admin-bar-notifications',
+		//account - special
+		'wp-admin-bar-get-apps', 'wp-admin-bar-next-steps', 'wp-admin-bar-help'
+	];
+
+	var notesTracksEvents = {
+		openSite: {
+			name: 'jetpack_masterbar_notifications_open_site',
+			properties: function( data ) { return { site_id: data.siteId, post_id: data.postId }; }
```

Comment:
This might also be more readable if move the function body to a new line (same goes for two cases bellow).

Reply:
Done. Changed on d2d94d956eb86c0266662e559e68f92a69ba679c